src/BusinessLogic/Resources/js/FooterController.js

```diff
@@ -32,10 +32,12 @@ var Packlink = window.Packlink || {};
                 closeSystemInfo
             );
 
+            document.addEventListener('keydown', closeSystemInfoOnEscape);
+
             debugModeCheckbox = templateService.getComponent('pl-debug-mode-checkbox', footer);
             debugModeCheckbox.addEventListener('click', debugModeCheckboxClickedHandler);
 
-            ajaxService.get(config.getStatusUrl, getDebugStatusHandler);
+            ajaxService.get(config.getDebugStatusUrl, getDebugStatusHandler);
```

Comment:
Is this Ok `.getDebugStatusUrl` instead of `.getStatusUrl`. What about when is not debug time?

Reply:
This is not a debug mode for us, this is a debug mode of the integration when we need to give support to a client. For this, each integration has a "debug" panel in the footer. This URL is needed for getting the status of the "debug mode". It can be turned on and off.